fastlane/Fastfile

```diff
@@ -83,6 +83,25 @@ platform :ios do
     )
   end
 
+  desc "Create an archive to be uploaded to the App Store"
+  lane :release do |options|
+    set_build_number(options)
+
+    carthage(
```

Comment:
I think we should probably just execute the `bootstrap.sh` script, which will run both carthage and npm for us.

Reply:
I'll add it to the beta lane as well

Reply:
We can add the bootstrap script later.